Source/Hat.cs

```diff
@@ -132,35 +144,45 @@ private void LogModLoadingState(Mod mod, bool loadState)
             }
         }
 
-        private void RemoveDuplicates()
+        private void DisableMods()
+        {
+            foreach (Mod mod in Mods)
+            {
+                ModConfig config = ConfigHelper.GetModConfig(mod.Info.Name, mod.Info.Version);
+                if (config.Disabled.HasValue && config.Disabled.Value == true)
+                    mod.IsEnabled = false;
+            }
+        }
```

Comment:
Not only the name of the method is not clear to what function it has, but I think this behaviour (handling configuration) should be done by the Mod class.

Reply:
Sure, sounds reasonable to me. I wasn't quite sure the best place to put this